src/shared/components/SearchBar/index.tsx

```diff
@@ -105,11 +105,13 @@ const SearchBar: React.FC<{
       },
       ...options,
     ];
+
     return options;
   };
 
   return (
     <AutoComplete
+      defaultActiveFirstOption
       className={`search-bar ${!!focused && 'search-bar__focused'}`}
```

Comment:
I guess the class name should be search-bar--focused here

Reply:
I think it is not even used anymore